lib/data/multipleBackendGateway.js

```diff
@@ -62,7 +63,28 @@ const multipleBackendGateway = {
         return client.delete(objectGetInfo, reqUids, callback);
     },
 
-    // checkHealth:
+    healthcheck: (log, callback) => {
+        const multBackendResp = {};
+        async.each(Object.keys(clients), (location, cb) => {
+            const client = clients[location];
+            if (client.clientType && client.clientType === 'scality') {
```

Comment:
Oooh, gotcha. I think I had things set up a little differently earlier so that I needed the check, but you're right, it's redundant here. Thx!